packages/components/src/DataList/components/DataListSearch/DataListSearch.css

```diff
@@ -1,14 +1,22 @@
+.wrapper {
+  display: flex;
+  align-items: center;
+}
+
 .searchInput {
   --transition-properties: var(--timing-base) ease-in-out;
-  --button-offset: calc(var(--space-largest) - var(--space-smallest));
+  --button-offset: calc(var(--space-largest) - var(--space-smaller));
 
   position: absolute;
   /* Inputs are off by 1 when put beside a button */
```

Comment:
Non-blocking - can this comment go away now since we're clearing the "1px" adjustment?

Reply:
Oh shoot, you right!